examples/xgboost/train_configs/config_train_exponential.json

```diff
@@ -0,0 +1,40 @@
+{
+    "data_path": "/media/ziyuexu/Data/HIGGS/HIGGS_UCI.csv",
```

Comment:
maybe we don't put these config in Github repo cuz it has your personal directory structure.

Reply:
agreed, change to "./dataset/HIGGS_UCI.csv"